dpe/src/response.rs

```diff
@@ -139,6 +139,9 @@ pub struct DeriveContextResp {
     pub resp_hdr: ResponseHdr,
     pub handle: ContextHandle,
     pub parent_handle: ContextHandle,
+    pub exported_cdi: [u8; MAX_EXPORTED_CDI_SIZE],
+    pub certificate_size: u32,
+    pub new_certificate: [u8; MAX_CERT_SIZE],
```

Comment:
@jhand2 These are defined as a separate variant for the response ABI shape.

Do you want to enforce that via build flag or do you want the struct shape to always be the same?

Reply:
The intention in the spec was that we would return a different one based on the input request flags. I left a related comment about this above.